pkg/operatorsource/configuring_test.go

```diff
@@ -23,15 +25,16 @@ func TestReconcile_NotConfigured_NewCatalogConfigSourceObjectCreated(t *testing.
 	controller := gomock.NewController(t)
 	defer controller.Finish()
 
-	nextPhaseWant := &marketplace.Phase{
+	nextPhaseWant := &v1.Phase{
```

Comment:
This is noise from an alias that we don't need. It seems like there's a bunch of that in this pr. Please remove it.